jenkins/manifests-update.jenkinsFile

```diff
@@ -29,7 +34,7 @@ pipeline {
             }
             steps {
                 script {
-                    withCredentials([usernamePassword(credentialsId: 'jenkins-github-bot-token', passwordVariable: 'GITHUB_TOKEN', usernameVariable: 'GITHUB_USER')]) {
+                    withSecrets(secrets: secret_github_bot){
```

Comment:
Sorry a bit confused. From where are we retrieving `secret_github_bot` name?

Reply:
Line10 of the same jenkinsfile where I added new entries.

Reply:
Oh missed it. Thanks!